content/docs/support-and-ops/ops-recipes/audit-logs-troubleshooting.md

```diff
@@ -47,3 +47,13 @@ cat /tmp/audit.log | jq '. | select(.user.username|test("joe."))'
 ```bash
 cat /tmp/audit.log | jq '. | select(.objectRef.name=="prometheus-prometheus-exporters-tls-assets")'
 ```
+
+## Using Loki
+
+Now all installations >= 19.1.0 have Loki instance deployed with audit logs included. So we can leverage on Lok to get the logs. Example of query:
```

Comment:
All AWS vintage installations have loki deployed,
but WCs logs are not collected yet.

When they will be collected, that will be for WCs >= 19.1.0, but we have some more checks to perform before we turn WC logs collection on.

Reply:
> Do we have a expected date for that? what about onprem, is Loki deployed there too? or planning?

I asked myself the same question, when I saw your original audit log documentation page and the Loki audit feature announcement on Slack (when to look where).

Reply:
For the moment we're focusing on Loki working well on AWS vintage, then we will add more cloudproviders.

WCs collection is under test on a few `anteater` clusters. We have some tuning to do on Loki for it to handle the load, and some cost estimation.

Onprem will probably be the most difficult one to add, because Loki relies on object storage and there's no standard object storage for capvcd.

Reply:
Ok I am adding a note with the supported envs